Lecture_05/Homework_05(1).py

```diff
@@ -0,0 +1,21 @@
+# 1. Створити програму, яка буде очікувати від користувача введення тексту 
+# і виведе інформацію по кожному надрукованому символу: 
+# це “число” + яке воно (парне, непарне), 
+# це “буква” + яка вона (велика чи маленька),
+# це “символ”
+
+s = input('Write me something, please: ')
+z = list(s)
```

Comment:
There is no need to transform string to a list since you can iterate through string without this transformation using for ... in ... like this: `for item in s:`